authentik/sources/ldap/sync/membership.py

```diff
@@ -34,7 +34,24 @@ def sync(self) -> int:
         )
         membership_count = 0
         for group in groups:
-            members = group.get("attributes", {}).get(self._source.group_membership_field, [])
+            if self._source.lookup_groups_from_user:
+                group_dn = group.get("dn", {})
```

Comment:
I noticed where you sometimes lookup an attribute called "distinguishedName" but you don't actually have to since the LDAP library your using always pulls the "dn" anyway. In Active Directory they always match as far as I can tell and "dn" is part of the LDAP spec.